examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>can  you rewrite nvtabular as NVTabular? May be rephrase <code>If you would like to learn more about NVTabular  library, ...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>